components/combobox/__tests__/__snapshots__/combobox.snapshot-test.jsx.snap

```diff
@@ -299,7 +300,8 @@ exports[`Base Custom Menu Item Open HTML Snapshot 1`] = `
 "<div class=\\"slds-form-element\\"><label class=\\"slds-form-element__label\\" for=\\"combobox-unique-id\\">Search</label>
   <div class=\\"slds-form-element__control\\">
     <div class=\\"slds-combobox_container\\">
-      <div class=\\"slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click ignore-react-onclickoutside slds-is-open\\" aria-expanded=\\"true\\" aria-haspopup=\\"listbox\\" aria-owns=\\"combobox-unique-id-listbox\\" role=\\"combobox\\">
+      <div class=\\"slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click ignore-react-onclickoutside slds-is-open\\" aria-describedby=\\"HJ4l8ZUm8_G-error-message\\" aria-expanded=\\"true\\" aria-haspopup=\\"listbox\\" aria-owns=\\"combobox-unique-id-listbox\\"
```

Comment:
`HJ4l8ZUm8_G-error-message` looks like a random assigned ID. this will fail tests.

Reply:
Good catch! I updated the test and passed in an ID.